### PR TITLE
Force a zypper refresh before installing

### PR DESCRIPTION
**Pull request by perlpunk**
Issue: https://progress.opensuse.org/issues/62009

Test run: https://openqa.opensuse.org/tests/1143664

### CODE REVIEW

File: lib/utils.pm [hunk 1]
```diff
@@ -4,11 +4,12 @@ use base Exporter;
 use Exporter;
 use strict;
 use testapi;
+use File::Basename qw(basename);
 
 our @EXPORT = qw(switch_to_x11 wait_for_desktop ensure_unlocked_desktop);
 
 sub switch_to_x11 {
-    my @hdd = split(/-/, get_required_var('HDD_1'));
+    my @hdd = split(/-/, basename get_required_var('HDD_1'));
```
Comment (okurz) [inline lib/utils.pm]: better if something like this is split into its own commit

File: tests/install/openqa_worker.pm [hunk 2]
```diff
@@ -5,8 +5,9 @@ use utils;
 
 sub run {
     diag('worker setup');
-    assert_script_run('zypper --no-cd --non-interactive --gpg-auto-import-keys in os-autoinst',   600);
-    assert_script_run('zypper --no-cd --non-interactive --gpg-auto-import-keys in openQA-worker', 600);
+    assert_script_run('zypper -n --gpg-auto-import-keys ref -f',                                  60);
```
Comment (okurz) [inline tests/install/openqa_worker.pm]: We don't have an automatic tidy check in this repo I assume but I doubt we need to keep the line like this. Looks weird if the timeout argument is separated by this many blanks. Also I suggest to just leave out the timeout and rely on the default which is 90s

Comment (perlpunk) [reply tests/install/openqa_worker.pm]: Oops, that was a leftover. Fixed.

File: lib/utils.pm (hunk 1 above)
Comment (perlpunk) [reply lib/utils.pm]: Ok, I put it in its own branch

**Comment by perlpunk** [comment]
Created #59 for the `basename`

File: lib/utils.pm (hunk 1 above)
Comment (okurz) [reply lib/utils.pm]: I am sorry if I did not make myself clear but I meant *just* **commit**, not its own PR or branch
